Service/SlidesInSlideDataCron.php

```diff
@@ -51,6 +51,14 @@ public function onCron(CronEvent $event)
     /** @var \Os2Display\CoreBundle\Entity\Slide $slide */
     foreach ($slidesOurType as $slide) {
 
+      /**
+       * We are running into issues, when we try to fetch data for more than ~20 slides concurrently. 
+       * This issued is cause by some security precautions from the external webserver. Therefore, 
+       * we added a tiny sleep delay between each call.
+       */
+      sleep(0.5);
+      echo "\nID: {$slide->getId()} ---------------------------------------------------------------------- \n";
```

Comment:
If we want to log data we should not do so using `echo`, `var_dump()` or the like. Instead: Use `$this->logger`. I also suggest using a bit more elaborate log message like `$this->logger("Processing slide {$slide->getId()")`.